code/scripting/api/libs/mission.cpp

```diff
@@ -337,6 +338,35 @@ ADE_FUNC(__len, l_Mission_SEXPVariables, NULL, "Current number of SEXP variables
 	return ade_set_args(L, "i", sexp_variable_count());
 }
 
+//****SUBLIBRARY: Mission/ShipRegistry
+ADE_LIB_DERIV(l_Mission_ShipRegistry, "ShipRegistry", nullptr, "The mission's ship registry: all ships parsed, created, or exited that the mission knows about", l_Mission);
```

Comment:
Should include that certain things, like ship-vanish, will not make ships sppear in the correct part of the mission log (or at all).

Reply:
I don't think that's necessary to say here, since the ship registry is completely independent of the mission log.  While "vanished" ships don't appear in the mission log, they are cleaned up just like any other exited ship.  The "exited" status in the ship registry covers all these cases.

Reply:
Ah, I see. I mixed that up then